package.json

```diff
@@ -3,9 +3,8 @@
   "version": "0.0.0-development",
   "description": "Convert DataTransfer object to a list of File objects",
   "main": "./dist/index.js",
-  "module": "./dist/es5/index.js",
-  "es2015": "./dist/es2015/index.js",
-  "typings": "./dist/index.d.ts",
```

Comment:
Just curious, has `typings` become a thing of the past?

Reply:
The `typings` field is an alias of `types`, so I used `types` instead of the alias. In a subsequent PR I am aiming to introduce the [`exports`](https://nodejs.org/api/packages.html#exports) field, which will also have to include it's own typings.